### PR TITLE
Fix kubeconfig fetching steps in readme

### PR DESCRIPTION
**Pull request by ashish-amarnath**
Signed-off-by: Ashish Amarnath <ashish.amarnath@gmail.com>

**What this PR does / why we need it**:
https://github.com/kubernetes-sigs/cluster-api-provider-docker/pull/44 changed the secret names. Updating the docs to be in sync.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

_Please confirm that if this PR changes any image versions, then that's the sole change this PR makes._

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -72,7 +72,7 @@ Make sure you have `kind` > 0.3.0 and `kubectl`.
 
 The kubeconfig is on the management cluster in secrets. Grab it and write it to a file:
 
-`kubectl get secrets -o jsonpath='{.data.kubeconfig}' kubeconfig-my-cluster | base64 --decode > ~/.kube/kind-config-my-cluster`
+`kubectl get secrets -o jsonpath='{.data.value}' my-cluster-kubeconfig | base64 --decode > ~/.kube/kind-config-my-cluster`
```
Comment (detiber) [inline README.md]: By chance can you use `.stringData.value` for the jsonpath and avoid needing to base64 decode it?

Comment (ashish-amarnath) [reply README.md]: Not sure I followed. 

But, here are the things that I tried, which all didn't work:
* `kc get secrets -o jsonpath='{.data.value.stringData}' my-cluster-kubeconfig`
* `kc get secrets -o jsonpath='{.stringData.value}' my-cluster-kubeconfig`
the resulting json doesn't have any non-encoded fields with kubeconfig

Comment (ashish-amarnath) [reply README.md]: And these are the keys at the top level json
```
kc get secrets -o json my-cluster-kubeconfig | jq keys
[
  "apiVersion",
  "data",
  "kind",
  "metadata",
  "type"
]
```

Comment (ncdc) [reply README.md]: I'm pretty sure `stringData` is a write-only field.

Comment (ashish-amarnath) [reply README.md]: From https://kubernetes.io/docs/concepts/configuration/secret/
> The stringData field is provided for convenience, and allows you to provide secret data as unencoded strings.

My interpretation is, you can create secrets by setting either `data` or `stringData`. If you are using `data` the content has to be encoded. Using `stringData`, allows you to use unencoded strings.

**Pull request merged**